cmd/cli/util.go

```diff
@@ -200,6 +234,16 @@ func getSupportedSmiForControllerPod(pod string, namespace string, restConfig *r
 		return nil, errors.Errorf("Error retrieving supported SMI versions for pod %s in namespace %s: %s", pod, namespace, err)
 	}
 
+	for smiApi, smiApiVersion := range smiSupported {
+		// smiApi looks like HTTPRouteGroup
+		// smiApiVersion looks like specs.smi-spec.io/v1alpha4
+		// leave out the API group and only keep the version after "/"
+		splitVersionInfo := strings.SplitN(smiApiVersion, "/", 2)
+		if len(splitVersionInfo) >= 2 {
+			smiSupported[smiApi] = splitVersionInfo[1]
+		}
+	}
+
 	return smiSupported, nil
```

Comment:
@shashankram With these changes, the API group is stripped out and only the version remains.

```
MESH NAME   MESH NAMESPACE   VERSION   ADDED NAMESPACES
osm         osm-system       v0.9.1    bookbuyer,bookstore,bookthief,bookwarehouse

MESH NAME   MESH NAMESPACE   SMI SUPPORTED
osm         osm-system       HTTPRouteGroup:v1alpha4,TCPRoute:v1alpha4,TrafficSplit:v1alpha2,TrafficTarget:v1alpha3

To list the OSM controller pods for a mesh, please run the following command passing in the mesh's namespace
	kubectl get pods -n <osm-mesh-namespace> -l app=osm-controller
```